bin/clean_output

```diff
@@ -0,0 +1 @@
+rm -r ./*/config*.dot ./*/*.dot.png
```

Comment:
This script isn't working for me.

rm: cannot remove './*/config*.dot': No such file or directory
rm: cannot remove './*/*.dot.png': No such file or directory

These are the files in the directory that I ran the script on:

abund.csv  config.Mouse.txt  config.Mouse.txt.dot  config.Mouse.txt.dot.png  expected.dot  expected.dot.png  README.md